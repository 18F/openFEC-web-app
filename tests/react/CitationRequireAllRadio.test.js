const rendered = require('react-test-renderer');
const React = require('react');
const ReactDOM = require('react-dom');
const CitationRequireAllRadio = require('../../static/js/legal/filters/CitationRequireAllRadio');

it('renders checkbox list', () => {
  const component = rendered.create(
    <CitationRequireAllRadio key="ao_citation_require_all" name="ao_citation_require_all" 
    handleChange={() => {}} value="" />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
})
