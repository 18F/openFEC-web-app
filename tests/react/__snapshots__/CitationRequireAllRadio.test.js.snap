// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders citation radio 1`] = `
<div>
  <label>
    If more than one citation is entered:
  </label>
  <input
    checked={true}
    id="citation_require_all_false"
    name="ao_citation_require_all"
    onChange={[Function]}
    type="radio"
    value={false}
  />
  <label
    htmlFor="citation_require_all_false"
  >
    Show cases citing any of them
  </label>
  <input
    checked={false}
    id="citation_require_all_true"
    name="ao_citation_require_all"
    onChange={[Function]}
    type="radio"
    value={true}
  />
  <label
    htmlFor="citation_require_all_true"
  >
    Show cases citing all of them
  </label>
</div>
`;
